subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/CrossVersionIntegrationSpec.groovy

```diff
@@ -24,13 +24,9 @@ import org.gradle.test.fixtures.file.TestNameTestDirectoryProvider
 import org.gradle.test.fixtures.maven.MavenFileRepository
 import org.junit.Rule
 import org.junit.runner.RunWith
-import spock.lang.Retry
 import spock.lang.Specification
 
-import static spock.lang.Retry.Mode.SETUP_FEATURE_CLEANUP
-
 @RunWith(CrossVersionTestRunner)
-@Retry(condition = { RetryConditions.onIssueWithReleasedGradleVersion(instance, failure) }, mode = SETUP_FEATURE_CLEANUP, count = 2)
```

Comment:
I am pretty sure we want to leave that retry in place. That is the current way to accept some flaky tests.